src/introduction.tex

```diff
@@ -46,15 +46,17 @@
 
 \subsubsection{Statement of the problem}
 
-Nowadays the industry increasingly relies on cloud technologies. More and more companies start or move their products to cloud environment.
-It has been observed that gradually institutions introduce dedicated teams for managing the costs of cloud operations or use third-parties for that purpose. Such solutions cause great savings for everyone.
-Unfortunately existing toolkit is either very primitive or targets billion dollar companies. Currently neither Amazon nor any third-party offer a tool that would allow flexible, cheap and quick insight into the operation costs for small or middle-sized organizations.
+The software industry increasing adopt public cloud as the primary method of running their infrastructure.
+
+As the cost of cloud increases, the companies invest in dedicated teams or tools to improve visibility and drive savings. Most of the current solution aims to reduce the cost of production workloads. However, systems for research and development have very different patterns, poorly addressed by existing tools. 
```

Comment:
very different -> distinct, might be a better wording but I'm not sure if this fits into what was meant to convey here, mark as resolved if it doesn't (optionally, find a way to express this divergence in some better way)

Reply:
cause I'm not sure what do we specifically mean here with `research`